.github/CODEOWNERS

```diff
@@ -6,21 +6,21 @@
 # Unless a later match takes precedence, they will be requested for review when someone opens a pull request.
 * @prometheus-community/helm-charts-admins
 
-/charts/alertmanager/ @monotek @naseemkullah
+/charts/alertmanager/ @naseemkullah
```

Comment:
It's just to get both aligned. Adding him on the chart requires a version bump anyhow. I could also keep it here as it was before.

Reply:
I would keep here for now only so that they can still help review PRs in the meantime. I opened https://github.com/prometheus-community/helm-charts/issues/153 to address this separately.

Reply:
updated